### PR TITLE
Serene 739-current state

### PR DESCRIPTION
**Pull request by seanli3**
Move client code to use react-scripts and implemented new UI designs

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,9 +0,0 @@
-{
```
Comment (lichaoir) [inline .babelrc]: So we don't babel anymore?

File: client/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (lichaoir) [inline client/.vscode/launch.json]: IDE specific configuration should be ignored.

File: .babelrc (hunk 1 above)
Comment (seanli3) [reply .babelrc]: we still do. This file has been moved into `./server`. And the client's babel config is in `client/package.json`

File: client/.vscode/launch.json (hunk 2 above)
Comment (seanli3) [reply client/.vscode/launch.json]: good catch, fixed

**Pull request merged**